WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StorePostViewModel.kt

```diff
@@ -57,6 +56,8 @@ class StorePostViewModel
     private val _onFinish = MutableLiveData<Event<ActivityFinishState>>()
     val onFinish: LiveData<Event<ActivityFinishState>> = _onFinish
 
```

Comment:
I wonder if we should make this volatile.. wdyt?

```suggestion

    @Volatile
```

Reply:
Suggestion accepted 👍